### ISSUE TITLE
Integration and deprecation of -m option in favor of JSON RPC

### ISSUE THREAD

**Issue by ann0see** [jamulussoftware/jamulus#3362]
It is worth discussing deprecating the -m option and/or integrating missing features it into JSON RPC. It's not a core feature of Jamulus. 

I think we had this discussion a while ago. Cc: @pljones 

_https://github.com/jamulussoftware/jamuluswebsite/pull/1019#discussion_r1730305176_

**Comment by pljones** [comment]
Yep, `--htmlstatus` should probably be marked deprecated in 3.12.0.  The client list is available from the server either over UDP (acting as a client, the way @softins's PHP does it) or using the JSON-RPC calls.

**Comment by ann0see** [comment]
The website already calls it deprecated today: https://jamulus.io/wiki/Running-a-Server#-m-or---htmlstatus

I think we should
- [ ] Update the help output in Jamulus to say it's deprecated
- [ ] Issue a warning in Jamulus if the -m option is invoked
- [ ] Remove all references to it on the website
- [ ] Write a discussion post with example how to get the all the outputs from the other APIs

**Comment by ann0see** [comment]
Ok. After checking the output it's really just the connected clients' names.

**Comment by ann0see** [comment]
Closing as announcement is now out: https://github.com/orgs/jamulussoftware/discussions/3508
